src/backend/expungeservice/models/charge_types/parking_ticket.py

```diff
@@ -23,4 +27,4 @@ def type_eligibility(self, disposition):
             )
 
     def hidden_in_record_summary(self, disposition):
-        return ChargeUtil.convicted(disposition)
+        return True
```

Comment:
Oh do we want to do this for traffic violations?

Reply:
We do not. Thanks. Some sort of copy-paste error...